src/mon/AuthMonitor.cc

```diff
@@ -1632,21 +1632,24 @@ bool AuthMonitor::prepare_command(MonOpRequestRef op)
 
       mds_cap_string += mds_cap_string.empty() ? "" : ", ";
       mds_cap_string += "allow " + cap;
-      if (path != "/") {
-	mds_cap_string += " path=" + path;
+
+      if (filesystem != "*" && filesystem != "all") {
```

Comment:
Can we name a filesystem "all?"

Reply:
It's a synonym, wasn't added in this commit. See -
https://github.com/ceph/ceph/pull/32581/files#diff-0df37d3bb6424ca661126c4889fb0bffL1640
https://github.com/rishabh-d-dave/ceph/commit/9c694ffc7eda5cf7979e5260db9119047ffada6a